xc/xc7/tests/dram/CMakeLists.txt

```diff
@@ -27,15 +23,17 @@ function(dram_tests dram_num_instances dram_mode outpad inpad enable_vivado_targ
 
 endfunction()
 
-# dram_tests(<dram_num_instances> <dram_mode> <syn_outpad_assert_usage> <syn_inpad_assert_usage> <enable_vivado_target>)
-dram_tests(4 32x1s 4 11 Y)
-dram_tests(8 32x1s 8 11 Y)
-dram_tests(2 32x1d 8 15 Y)
-dram_tests(4 32x2s 4 14 Y)
-dram_tests(2 64x1d 4 12 Y)
-dram_tests(4 64x1s 4 16 Y)
-dram_tests(2 128x1s 2 11 Y)
-dram_tests(1 128x1d 2 17 Y)
-dram_tests(1 256x1s 1 11 Y)
-dram_tests(1 32m 8 17 Y)
-dram_tests(1 64m 4 17 Y)
+# dram_tests(<dram_num_instances> <dram_mode> <syn_outpad_assert_usage> <syn_inpad_assert_usage> <blk_tl_slicem_assert_usage>)
+dram_tests(4 32x1s 17 18 1 Y)
```

Comment:
This still seems to have the last `Y)` bit?

Reply:
Sorry for that, it's removed now.